projects/llm-d/visualizations/llmd_inference/store/parsers.py

```
@@ -0,0 +1,264 @@
+    f"{artifact_dirnames.MULTITURN_BENCHMARK_DIR}/artifacts/multiturn_benchmark_job.logs",
+    f"{artifact_dirnames.GUIDELLM_BENCHMARK_DIR}/artifacts/guidellm_benchmark_job.logs",
+    f"{artifact_dirnames.PROMETHEUS_UWM_DUMP_DIR}/prometheus.tar.gz",
+]
+
+
+def parse_always(results, dirname, import_settings):
+    """Parsed even when reloading from the cache file"""
+    results.from_local_env = helpers_store_parsers.parse_local_env(dirname)
+
+
+def parse_once(results, dirname):
```

Ruff (0.15.1)

[warning] Unused function argument: `dirname`

(ARG001)